packages/react/src/components/ListItem/index.js

```diff
@@ -40,17 +43,29 @@ const ListItem = ({
         <Icon icon={leftIcon} />
       </div>
     )}
+
     <div>
-      <div
-        className={cx({
-          "ListItem-title": type !== LIST_ITEM_SIZES.MINI,
+      <UI
+        type={
+          type === LIST_ITEM_SIZES.MINI ? UI.TYPES.SubContent : UI.TYPES.Content
+        }
+        className={cx("ListItem-title", {
           "ListItem-subtitle": type === LIST_ITEM_SIZES.MINI,
         })}
       >
         {children}
-      </div>
-      {subtitle && <p className="ListItem-subtitle">{subtitle}</p>}
+      </UI>
+
+      {subtitle && (
+        <UI
+          type={LIST_ITEM_SIZES.MINI ? UI.TYPES.SubContent : UI.TYPES.Content}
```

Comment:
Shouldn't this condition be same as in L50?
It will always return true.

And if that's the case, shouldn't L50 and L60 become an arrow function?

Reply:
Oops yeah you're right

Reply:
Fixed! Alongside some proptypes